src/app/settings-embed/settings-embed.component.ts

```diff
@@ -0,0 +1,151 @@
+import { Component } from '@angular/core';
```

Comment:
Generally, it's not a good idea to just copy & paste all the code and create a different route to it.
We should just pass a parameter to the settings component to hide/show the elements.

Reply:
I know, but we need this quickly for xprize and altering the existing settings page was more work and would require more tests. I think we should do what you suggest as part of incorporating into the main app-v2 branch.